CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
```

Comment:
I was just linking to this file in the ach project. Should we have one file that's linked to in all projects?

Reply:
Yeah I guess I though the reference to ACH in the URL would be misleading.  Is there a place in the repo, we can have the code of conduct without a reference to a specific library?

Reply:
What about in the moov.io site? I'll offer to convert the markdown to html if we want. 

https://github.com/moov-io/moov-io

Reply:
That should work.